docs/developer/notification-center.md

```diff
@@ -0,0 +1,97 @@
+# Notification Center API
+
+> &#x26a0;&#xfe0f; Documentation in this directory is intended for internal use only. Any information contained here is unsupported.
+
+> For now, developers can use the Notification Centre via the VueX store. In the future, we will add an API for this, that will be fully supported as part of the UI Extensions API.
```

Comment:
nit: we switch between "Centre" and "Center". We should stick to one for consistency and use that throughout 🙂

Reply:
Indeed - I try to use the US spelling, but my better conscience is struggling with it